### PR TITLE
Enables display of commands with default settings in the `.nestrc `

### PR DESCRIPTION
**Pull request by steffengraber**
This simple solution separates the pager name and its options into a list. This allows the pager to be recognized correctly with the default settings in `.nestrc ` (`/page << /command (less -S)`).

This PR fix issue #1426

### CODE REVIEW

File: pynest/nest/lib/hl_api_helper.py [hunk 1]
```diff
@@ -501,15 +501,16 @@ def show_help_with_pager(hlpobj, pager=None):
         return
 
     try:
-        subprocess.check_call([pager, objf])
+        pagerl = pager.split(" ")
```
Comment (terhorstd) [inline pynest/nest/lib/hl_api_helper.py]: do we want to use shlex.split here? That would also handle spaces and quotes correctly, etc.
```suggestion
        import shlex
        pagerl = shlex.split(pager)
```

Comment (terhorstd) [reply pynest/nest/lib/hl_api_helper.py:504]: any opinion on this, anyone?

Comment (steffengraber) [reply pynest/nest/lib/hl_api_helper.py:504]: @terhorstd That looks good. I will be trying it out.

Comment (steffengraber) [reply pynest/nest/lib/hl_api_helper.py]: @terhorstd  Works for me. Changed it.

File: pynest/nest/lib/hl_api_helper.py [hunk 2]
```diff
@@ -499,7 +499,8 @@ def show_help_with_pager(hlpobj, pager=None):
         return
 
     try:
-        pagerl = pager.split(" ")
+        # pagerl = pager.split(" ")
```
Comment (terhorstd) [inline pynest/nest/lib/hl_api_helper.py:502]: remove commented code
```suggestion
```

**Comment by heplesser** [comment]
@steffengraber This one should be good to go once the gitignore conflict is resolved.

**Pull request merged**